components/suggestion/suggestion_box.jsx

```diff
@@ -709,16 +695,29 @@ export default class SuggestionBox extends React.PureComponent {
                     role='alert'
                     className='sr-only'
                 />
-                <QuickInput
-                    ref={this.inputRef}
-                    autoComplete='off'
-                    {...props}
-                    onInput={this.handleChange}
-                    onCompositionStart={this.handleCompositionStart}
-                    onCompositionUpdate={this.handleCompositionUpdate}
-                    onCompositionEnd={this.handleCompositionEnd}
-                    onKeyDown={this.handleKeyDown}
-                />
+                {renderTaggedInput ?
+                    <SearchSuggestionTaggedInput
```

Comment:
Instead of explicitly rendering `SearchSuggestionTaggedInput` here, would it be possible to do this with `QuickInput`'s `inputComponent` prop? That's already used to switch between an `input`, `textarea`, and `LocalizedInput` in various places where `SuggestionBox` is used.